### PR TITLE
Add more docker usage examples

### PR DESCRIPTION
**Pull request by eliecharra**
For #16

### CODE REVIEW

**Comment by sundowndev** [comment]
LGTM

**Review by sundowndev — APPROVED** [review]
LGTM

File: docs/installation.mdx [hunk 1]
```diff
@@ -94,6 +123,42 @@ $ docker run -t --rm \
...
+
+# With state stored on a s3 backend
+docker run -t --rm \
+  -v $(pwd):/app:ro \
+  -v ~/.driftctl:/home/.driftctl \
+  -e AWS_ACCESS_KEY_ID=XXXXXXXX \
+  -e AWS_SECRET_ACCESS_KEY=XXXX \
+  cloudskiff/driftctl scan --from tfstate+s3://my-bucket/path/to/state.tfstate
+
+# With multiples states
```
Comment (wbeuil) [inline docs/installation.mdx:145]: `multiple states`

File: docs/installation.mdx [hunk 2]
```diff
@@ -94,6 +123,42 @@ $ docker run -t --rm \
 
 `cloudskiff/driftctl:<VERSION_TAG>` run a specific driftctl tagged release
 
+## Usage examples
+
+```shell
+# With a local state
+docker run -t --rm \
```
Comment (wbeuil) [inline docs/installation.mdx:130]: Can you add for each docker statement the `$` to start a command

File: docs/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+# To specify AWS credentials
+$ AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=XXX driftctl scan
+# or using a named profile
+$ AWS_PROFILE=profile_name driftctl scan
+
+# With state stored on a s3 backend
+$ driftctl scan --from tfstate+s3://my-bucket/path/to/state.tfstate
+
+# With multiples states
```
Comment (wbeuil) [inline docs/usage.mdx:24]: `multiple states`

**Review by wbeuil — CHANGES_REQUESTED** [review]

**Review by wbeuil — APPROVED** [review]

**Pull request merged**